hawkbit-device-simulator/src/main/java/org/eclipse/hawkbit/simulator/WebSecurityConfiguration.java

```diff
@@ -28,6 +28,7 @@
  * Depending on the ~.auth.enabled property either basic authentication is enabled or access is granted for all requests.
  */
 @EnableWebSecurity
+@Order(SecurityProperties.BASIC_AUTH_ORDER + 1)
```

Comment:
The property `CONFIGURATION_ENABLED_PROPERTY` is controlling whether or not basic auth is enabled. So I'd make use of it and rather register only one `WebSecurityConfigurerAdapter`(with basic auth or without) than registering always two `WebSecurityConfigurerAdapter` instances and overwrite them by order

Reply:
Okay, I have changed it. The property "hawkbit.device.simulator.auth.enabled" is now controlling the selected web security configuration.